requirements.txt

```diff
@@ -2,3 +2,4 @@ sphinx==1.8.5
 Pygments>=2.0.1
 sphinx_bootstrap_theme
 transifex-client
+pbr
```

Comment:
I don't have much experience with sphinx but according to this https://github.com/opengisch/QField-docs/pull/100/checks?check_run_id=1509285590#step:7:289 it's not properly installed. I expected that there could be done a `pip install sphinxcontrib.video` but apparently it's not on pypi, so you could try `pip install git+https://github.com/sphinx-contrib/video.git` according to https://github.com/sphinx-contrib/video/issues/2